src/app/public/modules/progress-indicator/progress-indicator.component.ts

```diff
@@ -70,6 +81,11 @@ export class SkyProgressIndicatorComponent implements OnInit, AfterContentInit,
     return this._isPassive || false;
   }
 
+/**
+ * Specifies an observable of `SkyProgressIndicatorMessageType` that determines the status to
```

Comment:
I'm not sure what's going on, but when I serve locally, I'm getting the `SkyProgressIndicatorMessageType` text showing up within an `a` tag with `class` and `href` attributes.

![image](https://user-images.githubusercontent.com/7850105/80526559-eaf4d100-8960-11ea-83f1-b8cf9eb032f4.png)

@Blackbaud-SteveBrush, any idea what is going on here? Looking at this line in the code, the JSDoc comments clearly don't have that `a` tag.

Reply:
Appears to be a bug. Do you mind filing an issue in docs-tools?

Reply:
I just realized that no one followed up with Steve, and I created a docs-tool issue at https://github.com/blackbaud/skyux-docs-tools/issues/63.

Reply:
Resolving this conversation since it looks like we've got this in a state that avoids the weird `a` tag in the docs.